src/mobot_client/core/subscriber.py

```diff
@@ -327,11 +334,12 @@ def run_chat(self, break_on_stop=False, break_after=0):
         # t.start()
         self.logger.info("Now running MOBot chat...")
         while self._run:
-            try:
-                message = Message.objects.get_message()
-                if message:
-                    self.logger.info(f"Got message! {message}")
-                    self._process(message)
-            except Exception as e:
-                self.logger.exception("Exception getting message!")
-                time.sleep(5.0)
\ No newline at end of file
+            continue
+            # try:
+            #     message = Message.objects.get_message()
+            #     if message:
+            #         self.logger.info(f"Got message! {message}")
+            #         self._process(message)
+            # except Exception as e:
+            #     self.logger.exception("Exception getting message!")
+            #     time.sleep(5.0)
```

Comment:
I generally take a pretty hard-line stance on not committing commented-out code (mostly because git always persists historical information that we've since deleted). Was there a particular reason we wanted to keep this around?